origin.spec

```diff
@@ -46,15 +46,15 @@
 %endif
 %{!?make_redistributable: %global make_redistributable %{need_redistributable_set}}
 
-%if "%{dist}" == ".el7aos"
+%if 0%{?ose_build}
 %global package_name openshift
 %global product_name OpenShift
 %else
-%global package_name openshift
-%global product_name OpenShift
+%global package_name okd
+%global product_name OKD
```

Comment:
What are the rules on Product Name in general?

Reply:
not sure. If we could reuse the `openshift` naming scheme across the board, we wouldn't need the ose/ocp distinction in rpmbuild

Reply:
Yeah, let's reuse the openshift naming scheme across the board for now.  We call component pieces OpenShift and they are allowed to use the word.  It's only the top level that is officially called OKD for Reasons(tm) but we allow the use of the term to refer to the code base.